vertx-config/src/main/java/io/vertx/config/utils/FileSet.java

```diff
@@ -0,0 +1,134 @@
+package io.vertx.config.utils;
+
+import io.vertx.core.*;
+import io.vertx.core.impl.launcher.commands.FileSelector;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.config.spi.ConfigProcessor;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class to manage file set selected using a pattern.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public class FileSet {
```

Comment:
It is used by the other processors.

By moving it to impl, you break OSGi support as impl packages are not exported.